kwave/reconstruction/beamform.py

```diff
@@ -11,7 +11,8 @@
 
 from .shifted_transform import ShiftedTransform
 from .tools import make_time_vector, get_t0, get_origin_array, apodize
-from ..utils.conversion import scale_time, cart2pol
+from ..utils.conversion import cart2pol
```

Comment:
Relative imports are pain sometimes. Maybe let's refactor this file to avoid using them?

Reply:
All relative imports? i.e. also ```.tools import get_t0```? or just the double dots i.e. relative imports from parent packages?

Reply:
I meant all of them. Thought again about this. This change is probably out-of-scope for this PR. Let's refactor this in the future if you also think relative imports can be problematic